Makefile.am

```diff
@@ -312,7 +312,8 @@ $(BACKEND_TARGET): $(top_srcdir)/$(SCRIPTSDIR)/setup_compiler.sh | $(RESULTSDIR)
 	    --includedir "$(WDLC_INCLUDEDIR)" \
 	    --libdir "$(WDLC_LIBDIR)" \
 	    --srcdir "$(top_srcdir)" \
-	    --output "$(RESULTSDIR)"
+	    --output "$(RESULTSDIR)" \
+	    --local-packages "$(LOCAL_SITE_PACKAGES_DIR)"
```

Comment:
Prefer qualifying this since Python offers no equivalent convention. Perhaps:

* WDLC_PYTHONLOCALPACKAGEDIR

or:

* WDLC_PYTHON_LOCAL_PACKAGE_DIR

Reply:
Will do.